client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -129,6 +132,19 @@ export function getEnhancedTasks(
 						},
 					};
 					break;
+				case 'unlock_styles':
+					if ( ! isEnabled( 'limit-global-styles' ) || ! globalStylesInUse ) {
+						return;
+					}
+					taskData = {
+						title: translate( 'Activate advanced design customization tools' ),
+						isCompleted: ! shouldLimitGlobalStyles,
```

Comment:
When will this condition be met? It seems that the task will only become completed once the user upgrades to a paid plan, or am I missing something?

Reply:
> once the user upgrades to a paid plan

Correct! We need to show the step if the site is using custom global styles, and we need to show it as completed if we're not blocking the global styles in the frontend (because the site has the needed plan).

Reply:
Wouldn't it make more sense to mark the step as completed just by visiting it? Once the user ends up on the editor he won't be able to re-start the flow right? Then this step will never become completed?

Reply:
> Once the user ends up on the editor he won't be able to restart the flow right? 

I think My Home redirects to the launchpad until a first blog post is started, so users can definitely see the step again.

> the step here is to show the notice, not to upgrade the plan.

The step is meant to activate the ability to customize the styles which can be only completed by upgrading to a paid plan. Otherwise I think we should use a different copy instead of "Activate advanced design customization tools".